go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/stretchr/testify v1.8.4
 	github.com/temporalio/tctl-kit v0.0.0-20230328153839-577f95d16fa0
 	github.com/temporalio/ui-server/v2 v2.16.2
-	github.com/urfave/cli/v2 v2.23.6
+	github.com/urfave/cli/v2 v2.25.7
```

Comment:
i wonder if this is no longer the case for urfave/cli > v2.23.6 https://github.com/temporalio/cli/blob/d37f65cec42f708370ec06d3cf92088ce71eb827/Makefile#L20C147-L20C147

Since we now have e2e setup i'll likely rewrite this test with an actual workflow input https://github.com/temporalio/cli/blob/d37f65cec42f708370ec06d3cf92088ce71eb827/app/app_test.go#L127

Reply:
Hmm.. I just tested it and it seems to work. Should I delete the pinned dependency?